hadoop-gremlin/src/main/java/org/apache/tinkerpop/gremlin/hadoop/Constants.java

```diff
@@ -73,6 +73,7 @@ private Constants() {
     public static final String GREMLIN_SPARK_PERSIST_STORAGE_LEVEL = "gremlin.spark.persistStorageLevel";
     public static final String GREMLIN_SPARK_SKIP_PARTITIONER = "gremlin.spark.skipPartitioner"; // don't partition the loadedGraphRDD
     public static final String GREMLIN_SPARK_SKIP_GRAPH_CACHE = "gremlin.spark.skipGraphCache";  // don't cache the loadedGraphRDD (ignores graphStorageLevel)
+    public static final String GREMLIN_SPARK_DONT_DELETE_NONE_EMPTY_OUTPUT = "gremlin.spark.dontDeleteNonEmptyOutput"; // don't delete the output if it is not empty
```

Comment:
Nit: I think there is a typo here. It should probably be GREMLIN_SPARK_DONT_DELETE_NON_EMPTY_OUTPUT

Reply:
Ack